tests/avocado/avocado_qemu/test.py

```diff
@@ -379,6 +386,49 @@ def add_image(self, path, username=None, password=None, snapshot=True,
         if password is not None:
             self.password = password
 
+    def cloudinit(self, hostname=None, password=None):
```

Comment:
As discussed, it may be a good thing to also include a username, so that the default experience (username/password) is the same across the board.

The user/password would cascade from the parameter system (including defaults), into cloudinit, and into the session.  Users would know how to log into misc VMs, no matter their flavor (as long as they are cloudinit compliant).

Reply:
yep, working on that